tests/jobscript/11-shell-ksh.t

```diff
@@ -25,7 +25,7 @@ set_test_number 5
 install_suite "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
 run_ok "${TEST_NAME_BASE}-validate" \
     cylc validate "${SUITE_NAME}" --set=KSH="${KSH}"
-sed -in 's/^.* WARNING - //p' "${TEST_NAME_BASE}-validate.stderr"
+sed -in 's/^.*WARNING - //p' "${TEST_NAME_BASE}-validate.stderr"
```

Comment:
This change is not really related to the fix.

Reply:
Dammit, missed one!